src/context/StationsContext.js

```diff
@@ -17,11 +22,26 @@ export const StationsContextProvider = ({ children }) => {
     refreshRadioStations();
   }, []);
 
+  function getRandomRadtioStation() {
+    const randomInt = getRandomInt(0, radioStationsListLS.length);
+    return radioStationsListLS[randomInt];
+  }
```

Comment:
You could create a pure function out of this by moving it out of the component and passing the array as a param

```suggestion
  function getRandomItem(items) {
    const randomInt = getRandomInt(0, items.length);
    return items[randomInt];
  }
```